src/HotChocolate/Core/src/Types.NodaTime/Extensions/ISchemaBuilderExtensions.cs

```diff
@@ -0,0 +1,26 @@
+using HotChocolate;
+using HotChocolate.Types.NodaTime;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class ISchemaBuilderExtensions
+    {
+        public static ISchemaBuilder AddNodaTime(this ISchemaBuilder schemaBuilder)
+        {
+            return schemaBuilder
+                .AddType<DateTimeZoneType>()
```

Comment:
Shouldn't this be `IRequestExecutorBuilder` now?

Reply:
We should kick this out completely for 11. We cannot introduce IRequestBuilder here since we then get a dependency on the execution engine.

What I want to do for 11 is to create a new interface `IGraphQLBuilder`.  `IRequestExecutorBuilder` will implement `IGraphQLBuilder` but the new builder interface will live in a new HotChocolate.Extensions library. This will allow for extensibility without the need to reference the whole execution engine.

So, with this in mind, either lets keep this class as is and then deprecate it or lets kick it out.

Reply:
Personally I think it should go into 12 then, once the ground work for the extensions has been done. I don't think this is something mission critical that needs to go into 11 no matter what. People can just use the old library.

Reply:
I feel the same. will remove it then for now.